client/app/components/file-uploader/component.js

```diff
@@ -1,61 +1,25 @@
 import Ember from 'ember';
 //import ENV from 'analytics-dashboard/config/environment';
 
-function getToken() {
-    const session = window.localStorage['ember_simple_auth:session'];
-    if (session) {
-        const token = JSON.parse(session)['authenticated'];
-        if ('attributes' in token) {
-            return token['attributes']['accessToken'];
-        }
-        return token;
-    }
-}
-
 export default Ember.Component.extend({
 
     actions: {
 
-        uploadFile: function(target) {
+        uploadFile: function(ev) {
 
-            const saveParameter = this.attrs.saveParameter;
             const reader = new FileReader();
-            const f = event.target.files[0];
-            const uri = "https://files.osf.io/v1/resources/h8d72/providers/osfstorage/?kind=file&name=" + f.name;
```

Comment:
Can you change this url depending on the current OSF backend? Right now file uploads will only work with production as the backend.